### ISSUE TITLE
Multitude of errors

### ISSUE THREAD

**Issue by Shizzleshizz** [philippnormann/nvidia-sniper#18]
Followed Directions (sorry, new to this). Received following errors:

C:\Users\usr\nvidia-sniper>pipenv run python -m sniper
Traceback (most recent call last):
  File "c:\users\usr\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\usr\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\usr\nvidia-sniper\sniper\__main__.py", line 6, in <module>
    import aiohttp
  File "C:\Users\usr\.virtualenvs\nvidia-sniper-1gkHHUZV\lib\site-packages\aiohttp\__init__.py", line 6, in <module>
    from .client import BaseConnector as BaseConnector
  File "C:\Users\usr\.virtualenvs\nvidia-sniper-1gkHHUZV\lib\site-packages\aiohttp\client.py", line 32, in <module>
    from . import hdrs, http, payload
  File "C:\Users\usr\.virtualenvs\nvidia-sniper-1gkHHUZV\lib\site-packages\aiohttp\http.py", line 7, in <module>
    from .http_parser import HeadersParser as HeadersParser
  File "C:\Users\usr\.virtualenvs\nvidia-sniper-1gkHHUZV\lib\site-packages\aiohttp\http_parser.py", line 15, in <module>
    from .helpers import NO_EXTENSIONS, BaseTimerContext
  File "C:\Users\usr\.virtualenvs\nvidia-sniper-1gkHHUZV\lib\site-packages\aiohttp\helpers.py", line 43, in <module>
    import async_timeout
ModuleNotFoundError: No module named 'async_timeout'

**Comment by philippnormann** [comment]
Hey @Shizzleshizz, have you installed the project dependencies using `pipenv install`, prior to launching the bot?

**Comment by Shizzleshizz** [comment]
Hey! I have ran that successfully! Followed the step by step. Used powershell and CMD. At first powershell wasn’t able to find pip in the PATH and got that fixed, then ran into all these once I tried to run it.  Tried deleting and reinstalling everything too with no luck. 

I’ll have to check it all out again tonight to see if I’m running into the same issues. I am running a new clean copy of windows. Could that be it?
